src/main/resources/project/forms/deployment/CreateVMForm.xml

```diff
@@ -151,7 +151,7 @@
     <formElement>
         <label>Resource Port:</label>
         <property>resource_port</property>
-        <value/>
+        <value>7800</value>
```

Comment:
You do not have to specify the port number. EF will default it to 7800 if none is specified.